platform/mellanox/iproute2/scripts/get_sdk_patches.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/python3
+
+'''
+Script to download SDK iproute2 patches.
+
+If we build SONiC with SDK debians being taken from github, need to downlaod patches from github, usage will be:
+./get_sdk_patches.py -i <iproute2 version> -s <SDK version>
+
+If we build SONiC with SDK compiled from sources, need to take patches from SDK sources, usage will be:
+./get_sdk_patches.py -i <iproute2 version> -u <SDK sources url> 
+'''
+import argparse
+import requests
+import sys
+import os
+import subprocess
+from zipfile import ZipFile
+
+SDK_GITHUB_URL_BASE = 'https://raw.githubusercontent.com/Mellanox/Spectrum-SDK-Drivers/'
```

Comment:
Please pass this as argument to script from Makefile. Lets not hardcode in the script

Reply:
Done